maze-info.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index abb1d02..76aba63 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -3,9 +3,14 @@ name = Maze
+ exec = sugar-activity3 activity.MazeActivity
+ icon = activity-maze
+ activity_version = 29
++release_date = 2018-04-12
+ license = GPLv3+
++metadata_license = CC0-1.0
++update_contact = tch@sugarlabs.org
+ bundle_id = vu.lux.olpc.Maze
+ summary = Good at finding your way out of hard situations? Let's see! Try to make your way through an increasingly difficult path, or you can also play with a friend!
+ max_participants = 6
+ repository = https://github.com/sugarlabs/maze-activity
+ tags = Game
++description = A simple maze game for the Sugar learning environment
```

Comment:
Swap summary and description texts.

Reply:
Also, keep in mind collaboration is disabled in sugarapp. So remove the "play with friend" part. Try to mix the descriptions we have in activities.sugarlabs.org and the wiki to present a good description.

Reply:
resolved in ed9beab